README.md

```diff
@@ -294,6 +302,24 @@ Effectively, skipped rules are treated as if they don't exist.
 not in the organization that owns the repository where the rules appear. In
 this case, `policy-bot` must be installed on all referenced organizations.
 
+#### Update Merges
+
+For a commit on a branch to count as an "update merge" for the purpose of the
+`ignore_update_merges` option, the following must be true:
+
+1. The commit must have exactly two parents
+2. The commit must have the `committedViaWeb` property set to `true`
+3. One parent must exist in the last 100 commits on the target branch of the
+   pull request
+
+These will all be true after updating a branch using the UI, but historic
+merges on long-running branches or merges created with the API may not be
+ignored. If this happens, you will need to reapprove the pull request.
+
+Note that `policy-bot` cannot detect if an update merge contains any merge
```

Comment:
that last sentence sounds a bit dangerous. i wonder how many of the use cases motivating this feature could be addressed with just requiring 2 approvals and allowing authors/contributors to be approvers

Reply:
I think the main use case is when you get approval, and then someone else merges another branch first, so now you have to update your branch and get approval again. That doesn't change based on the approval conditions.

For the case where you want to ignore other contributors who click update, then the double approval approach could work.